platforms/documentation/docs/src/docs/userguide/optimizing-performance/configuration_cache.adoc

```diff
@@ -197,6 +197,29 @@ or in a `gradle.properties` file:
 org.gradle.configuration-cache.max-problems=5
 ----
 
+[[config_cache:usage:parallel]]
+=== Enabling parallel configuration caching
+
+Configuration cache storing and loading are done sequentially by default. Parallel storing and loading provide
+better performance, however not all builds are compatible with it.
+
+To enable parallel configuration caching on the command line:
+
+----
+❯ gradle -Dorg.gradle.configuration-cache.parallel=true
+----
+
+or in a `gradle.properties` file:
+
+[source,properties]
+----
+org.gradle.configuration-cache.parallel=true
+----
+
+The parallel configuration caching feature is _incubating_, as not all builds are guaranteed to work correctly.
```

Comment:
❌   We are saying that this feature is incubating, but AFAICT we don't issue incubation messages, to which users are used to.

I strongly feel we should use an `IncubationLogger` instead of the debug messages. We can lift the incubation later even with keeping the feature opt-in. But without the incubation message, it looks like opt-ing in into Parallel CC is production ready.

Reply:
Thanks, `IncubationLogger` is being invoked now